release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -279,6 +279,16 @@ For more information, see xref:../networking/hardware_networks/about-sriov.adoc#
 [id="ocp-4-14-storage"]
 === Storage
 
+[id="ocp-4-14-storage-device-selector"]
+==== Support for OR logic in LVMS
+
+With this release, the LVM Cluster Custom Resource (CR) provides `OR` logic in the `deviceSelector` section. In previous releases, specifying the `paths` setting for device paths used `AND` logic only. With this release, you can also specify `optionalPaths`, which supports `OR` logic. For more information, see the CR examples in xref:../storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc#persistent-storage-using-lvms[Persistent storage using logical volume manager storage].
+
+[id="ocp-4-14-storage-lvms-ext4-support"]
+==== Support for ext4 in LVMS
+
+With this release, the LVM Cluster Custom Resource (CR) provides support for the `ext4` filesystem with the `fstype` setting under `deviceClasses`. The default filesystem is `xfs`. For more information, see the CR examples in xref:../storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc#persistent-storage-using-lvms[Persistent storage using logical volume manager storage].
```

Comment:
"LVM Cluster Custom Resource (CR)"

I'm unsure why "Cluster" is capitalized? According to the SSG, "custom resource" is the correct case:

![Screenshot from 2023-09-27 09-49-28](https://github.com/openshift/openshift-docs/assets/57954076/b91be225-a689-4555-a91d-553b9d608f3e)

Reply:
`deviceClasses` subdirectory?

Reply:
Done